src/index.ts

```diff
@@ -99,13 +117,14 @@ export class VisualObserver {
       };
     }
 
-    // This was previously document.scrollingElement?
-    const x = left + root.scrollLeft;
-    const y = top + root.scrollTop;
+    // The biggest bottleneck Im seeing is accessing these `root` properties
```

Comment:
The biggest perf bottleneck im seeing is accessing these on every element refresh